src/main/java/org/smartregister/pathevaluator/task/TaskHelper.java

```diff
@@ -45,6 +45,11 @@ public void generateTask(Resource resource, Action action, String planIdentifier
 		task.setExecutionEndDate(getDateTime(action.getTimingPeriod(), false));
 		task.setAuthoredOn(DateTime.now());
 		task.setLastModified(DateTime.now());
+        //TODO : Generate task according to defaultBusinessStatus
+//		if(action.getDynamicValue().getExpression().getName().equals("defaultBusinessStatus")) {
+//		task.setBusinessStatus();
+//
+//		}
```

Comment:
Complete this implementation and add a unit test

Reply:
Done